README.md

```diff
@@ -71,8 +71,7 @@ It is best to create command shorthands for the fuzzing configurations you will
 
 ```
 # Force the use of Clang for C++ builds.
-build --action_env=CC=clang
-build --action_env=CXX=clang++
+build --repo_env=CC=clang
```

Comment:
Is this change necessary?

If the intent is to highlight GCC support in the README.md file, I would prefer to still recommend using clang by default, because it is a simpler setup. We can still mention that GCC is supported for non-fuzzer configurations.

Reply:
It is not necessary to achieve GCC compatibility, it just makes switching compilers more cache-friendly in general. Using `action_env` instead of `repo_env` means that the environment for all actions is changed by this config, rather than just that of repository rules (which are responsible for auto-configuring the C/C++ toolchain). 
Since [`CXX` is not read by Bazel](https://github.com/bazelbuild/bazel/blob/f141e149ac3210e417b31aec21afed267cdbb092/tools/cpp/cc_configure.bzl#L147), the second line appears to be entirely redundant.

If you prefer, I can drop this commit and potentially submit it as a separate PR.

Reply:
> It is not necessary to achieve GCC compatibility, it just makes switching compilers more cache-friendly in general. Using `action_env` instead of `repo_env` means that the environment for all actions is changed by this config, rather than just that of repository rules (which are responsible for auto-configuring the C/C++ toolchain).

Interesting, I did not realize these vars are only used during toolchain configuration, and not inside each C++ compilation action.

> Since [`CXX` is not read by Bazel](https://github.com/bazelbuild/bazel/blob/f141e149ac3210e417b31aec21afed267cdbb092/tools/cpp/cc_configure.bzl#L147), the second line appears to be entirely redundant.

Can we assume this will never change? If the standard is to configure both variables, I'd say we should follow that and give the Bazel team the liberty to add CXX support in the future.

> 
> If you prefer, I can drop this commit and potentially submit it as a separate PR.

Yeah, that sounds like a better approach.